### PR TITLE
[XBMC Action][XMPP Action][xPL Action] Correct linter errors only

### PR DESCRIPTION
**Pull request by bgilmer77**
Signed-off-by: Brad Gilmer <brad@gilmer.tv>

### CODE REVIEW

File: bundles/action/org.openhab.action.xmpp/README.md [hunk 1]
```diff
@@ -4,37 +4,37 @@ This set of actions allows you to send a message to an XMPP user or multi-user c
...
+|--------------|---------------|-:-:------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```
Comment (9037568) [inline bundles/action/org.openhab.action.xmpp/README.md]: This is broken.

**Comment by bgilmer77** [comment]
@Confectrician, Jerome can you please take a look at this?  I think the lines in this table are so long that they are blowing up my Atom editor.  Can you fix this?

**Comment by Confectrician** [comment]
Did you try improving the table with https://www.tablesgenerator.com/markdown_tables ?
This worked for me always.
I am on mobile now. Will have a further look tommorow.

**Comment by bgilmer77** [comment]
Yes, I cut the badly formed table from the article, put it into the tablesgenerator.com website, fixed everything and then pasted it back where the original table was located.

**Comment by 9037568** [comment]
It appears to me this is just having colons in the wrong place.

If I change the text from the existing:
````
|-:-:------|
````

To this:
````
|:---------:|
````

The table works again.

**Comment by Confectrician** [comment]
I think @9037568 is right.

Take this one @bgilmer77, it should work.

```
| Property     | Default       | Required | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
|--------------|---------------|:--------:|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| username     |               |          | The username of the XMPP account used by openHAB. Most services will require that you use only the localpart of the account's JID. For example if your accounts JID is `myuser@example.org`, then only configure `myuser`.                                                                                                                                                                                                                                                                                                                                                                    |
| password     |               |          | The password of the XMPP account used by openHAB                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
| servername   |               |          | The XMPP service to use, e.g. `jabber.de`. A list of public XMPP services can be found at https://xmpp.net/directory.php                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
| securitymode | `disabled`    |    No    | The Security mode used for the XMPP connection. Can be either `required` or `disabled`. Defaults to `disabled`, which means that TLS will not be used.  Warning: If you change this to non-disabled, then you must make sure that your TLS server certificate can be validated, otherwhise the connection will fail.                                                                                                                                                                                                                                                                          |
| tlspin       |               |          | The TLS Pin used to verify the XMPP service's certificate. Set this in case openHAB's default `SSLContext` is unable to verfiy it (e.g. because the XMPP service uses a self-signed certificate). The PIN value is bascially the hash of the certificate in hexadecimal notation. You have to set `securitymode` to `required` to enable TLS for XMPP connections. For information on how to generate the PIN visit https://github.com/Flowdalic/java-pinning .  Example: `tlspin=CERTSHA256:83:F9:17:1E:06:A3:13:11:88:89:F7:D7:93:02:BD:1B:7A:20:42:EE:0C:FD:02:9A:BF:8D:D0:6F:FA:6C:D9:D3` |
| proxy        |               |          | The XMPP Proxyserver to use, e.g. `gmail.com`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
| port         | `5222`        |          | The server port to use                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
| consoleusers |               |          | A comma-separated list of users that are allowed to use the XMPP console                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
| chatroom     |               |          | The multi user chat to join, e.g. `openhab@chat.example.com`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
| chatnickname | `openhab-bot` |    No    | The nickname used in the multi-user chat                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
| chatpassword |               |          | The password required to join the multi user chat                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
```
---

Did you edit the table by hand?

I just copy the old table data, paste it and generate new.
Worked always so far.

![2018-01-04_20-46-48](https://user-images.githubusercontent.com/7161177/34581463-c0aafbfa-f190-11e7-9389-a557a1094c1f.gif)

To mark everything for copying in the box, just double click .

**Comment by bgilmer77** [comment]
Guys, I do not seem to be able to fix this table properly given its extended length.  I am using Ubuntu Desktop with Atom.  Somehow when I paste the table into Atom, it does not keep the formatting from the websites.  I give up.  What shall we do?  Shall we close this PR and let someone else fix it?

File: bundles/action/org.openhab.action.xmpp/README.md [hunk 2]
```diff
@@ -4,37 +4,38 @@ This set of actions allows you to send a message to an XMPP user or multi-user c
...
+|--------------|---------------|---------:|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```
Comment (Confectrician) [inline bundles/action/org.openhab.action.xmpp/README.md]: Can you type in the `:` by hand?

So we get from
`|---------:|`
to
`|:---------:|`

I think that's the only thing thats missing here.

**Comment by bgilmer77** [comment]
Yes, I can do that. Thank you.

**Comment by bgilmer77** [comment]
That was a pain!  I think it is okay now.  Could someone please check?  What is the purpose of the colons in a table?  I never saw that before.

**Comment by 9037568** [comment]
Looks good now. Thanks!

The colons set the alignment of the columns (see [this cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables))

**Comment by bgilmer77** [comment]
Very helpful 👏. Now I know what the colons are for. Thank you.

**Pull request merged**